### DISCUSSION TITLE
I'M BLOCKED FROM CREATING AN ACCOUNT AND YOU GUYS

### DISCUSSION THREAD

**Discussion by plmts** [vercel/community#3478] [Help]
### Summary

Ok, it's my second time here trying to find a solution to my problem. I've already send you emails explaining what's goind on and you guys are asking the same questions that i've already answer. Seems like you don't ready my messeges, so i'm trying again. Let's go step by step:

>>>>
First step: I go to the page to create an account. The page asks me choose btw "hobby" or "pro". I choose hobby;
Second step: After that, the page asks for my name. I type my name (and any other name do the same problem);
Third step: There's a button "continue". I click on it to continue.
Then i receave the msg: "You have been blocked from creating an account. Request id: 3de274f0-9c19-4c95-b176-6417005d3677"
<<<<

So, i dont have the chance to choose username or to type my email, i'm not able to do anything. I'm just blocked.
In every email i sent to your support email, you keep asking me the username and the email i'm trying to register, but there is no one because i cant even have a chance to choose an email or username. And i told this in dozens emails i sent. I even send a video showing what is going on, and you guys keep asking the username and the email i'm trying to register. 

So, what can I do (besides aswering the same emails with the same answer). 

ps.: I bet you're gonna tell me to send another email.

https://github.com/vercel/community/assets/90912163/1b00ec0f-048e-4582-a0a8-6a09ad239479

### Steps to Reproduce

i thing you can't reproduce this error because as far i know, its happening to me

**Comment by amyegan** [comment]
Hey @plmts. I'm sorry you're facing this frustration. The reason we ask that you send an email is so we can keep your account details and contact information private. If you let me know your case number, I can follow up with the team 🙏

**Reply by plmts** [reply]
Case number: 00148684
